app/services/RegistrationService.scala

```diff
@@ -96,6 +83,28 @@ class RegistrationService {
     Future.fromTry(userAnswers)
   }
 
+  private def getEuRegistrationDetails(registration: Registration): Seq[EuOptionalDetails] = {
+    registration.euRegistrations.zipWithIndex map {
+      case (euVatRegistration: EuVatRegistration, _) =>
+        EuOptionalDetails(euCountry = euVatRegistration.country, None, None, None, None,
```

Comment:
can we make these a little more readable by adding the parameter your assigning it to for all values?

Reply:
Fixed